enterprise/admin/views.py

```diff
@@ -828,6 +828,7 @@ def _enroll_users(
             notify: Whether to notify (by email) the users that have been enrolled
         """
         pending_messages = []
+        paid_modes = ['verified', 'professional']
```

Comment:
Can we use [paid_modes_for_course](https://github.com/edx/edx-platform/blob/08877db85836d9630ef39483a5bc8e294375af16/common/djangoapps/course_modes/models.py#L313) instead of hardcoded values here?

Reply:
Let's try not to create this coupling with platform.